test/local_sensitivity/sde.jl

```diff
@@ -298,29 +296,31 @@ end
 
   @info res_sde_p
 
-  res_sde_u02, res_sde_p2 = adjoint_sensitivities(sol_sde,EulerHeun(),dg!,Array(t)
-      ,dt=tend/1e6,adaptive=false,sensealg=BacksolveAdjoint(noise=false))
-
-  @info res_sde_p2
-
-  @test isapprox(res_sde_p, res_sde_p2, rtol = 1e-5)
-  @test isapprox(res_sde_u0, res_sde_u02, rtol = 1e-5)
-
-  res_sde_u02, res_sde_p2 = adjoint_sensitivities(sol_sde,EulerHeun(),dg!,Array(t)
-      ,dt=tend/1e6,adaptive=false,sensealg=BacksolveAdjoint(noise=DiffEqSensitivity.ZygoteNoise()))
-
-  @info res_sde_p2
-
-  @test_broken isapprox(res_sde_p, res_sde_p2, rtol = 1e-5)
-  @test isapprox(res_sde_u0 ,res_sde_u02, rtol = 1e-5)
-
-  res_sde_u02, res_sde_p2 = adjoint_sensitivities(sol_sde,EulerHeun(),dg!,Array(t)
-      ,dt=tend/1e6,adaptive=false,sensealg=BacksolveAdjoint(noise=DiffEqSensitivity.ReverseDiffNoise()))
-
-  @info res_sde_p2
-
-  @test isapprox(res_sde_p, res_sde_p2, rtol = 1e-10)
-  @test isapprox(res_sde_u0 ,res_sde_u02, rtol = 1e-10)
+  ### The following tests are not needed anymore here, as for scalar SDEs it is sufficient to compute the vjps for the dsigma/dp terms
```

Comment:
then delete them?

Reply:
Looks to me like it's still a useful test though. It's stressing a different part of the adjoint method. Technically it no user should be using this here because it is a scalar SDE, but it's hitting a code path that would be used on larger examples.